narwhals/_spark_like/expr.py

```diff
@@ -148,22 +138,12 @@ def _Window(self) -> type[Window]:  # noqa: N802
     def _sort(
         self, *cols: Column | str, descending: bool = False, nulls_last: bool = False
     ) -> Iterator[Column]:
-        """Sort one or more columns.
-
-        Arguments:
-            *cols: One or more columns, or a `WindowInputs` object - where `order_by` will be used.
-            descending: Sort in descending order.
-            nulls_last: Place null values last.
-
-        Yields:
-            Column expressions, in order of appearance in `cols`.
-        """
         F = self._F  # noqa: N806
         mapping = {
-            ASC_NULLS_FIRST: F.asc_nulls_first,
-            DESC_NULLS_FIRST: F.desc_nulls_first,
-            ASC_NULLS_LAST: F.asc_nulls_last,
-            DESC_NULLS_LAST: F.desc_nulls_last,
+            (False, False): F.asc_nulls_first,
+            (False, True): F.desc_nulls_first,
+            (True, False): F.asc_nulls_last,
+            (True, True): F.desc_nulls_last,
         }
         sort = mapping[(descending, nulls_last)]
```

Comment:
hope this isn't too controversial, personally i find that having these as constants introduces more indirection than is necessary, it's simpler and clearer to just have the values here, the next line just does `mapping[(descending, nulls_last)]` anyway so it's clear what each refers to